src/main/java/stanhebben/zenscript/expression/ExpressionJavaLambdaSimpleGeneric.java

```diff
@@ -114,28 +123,22 @@ public void compile(boolean result, IEnvironmentMethod environment) {
                     bridge.load(org.objectweb.asm.Type.getType(method.getParameterTypes()[i]), ++i);
                 }
             }
-
+            
             bridge.invokeVirtual(clsName, method.getName(), descriptor);
             bridge.returnType(org.objectweb.asm.Type.getReturnType(method));
             bridge.end();
         }
-
+        
         environmentMethod.createConstructor(cw);
         environment.putClass(clsName, cw.toByteArray());
-    
+        
         // make class instance
         environment.getOutput().newObject(clsName);
         environment.getOutput().dup();
-        final String[] arguments = environmentMethod.getCapturedVariables().stream()
-                .map(SymbolCaptured::getEvaluated)
-                .peek(expression -> expression.compile(true, environment))
-                .map(Expression::getType)
-                .map(ZenType::toASMType)
-                .map(Type::getDescriptor)
-                .toArray(String[]::new);
+        final String[] arguments = environmentMethod.getCapturedVariables().stream().map(SymbolCaptured::getEvaluated).peek(expression -> expression.compile(true, environment)).map(Expression::getType).map(ZenType::toASMType).map(Type::getDescriptor).toArray(String[]::new);

```

Comment:
Why the reformat instead of keeping it chopped?

Reply:
Sorry, this is just a mistake because i forge to sync code style to ide at first and forge to rollback some changes. I'll settle that later